src/machine.rs

```diff
@@ -238,8 +240,21 @@ impl<'m> Machine<'m> {
     pub async fn stop(&mut self) -> Result<(), Error> {
         let vm_id = self.config.vm_id();
         trace!("{vm_id}: Killing VM...");
-        let child = heim::process::get(self.pid.try_into()?).await?;
-        child.kill().await?;
+
+        let pid = self.pid;
+        let killed = task::spawn_blocking(move || {
+            let sys = System::new_all();
+            match sys.process(Pid::from(pid)) {
+                Some(process) => Ok(process.kill()),
+                None => Err(Error::ProcessNotRunning(pid)),
+            }
```

Comment:
If you only care about one process to check if it's alive, better do this:

```suggestion
            let sys = System::new();
            if sys.refresh_process_specifics(Pid::from(pid), ProcessRefreshKind::new()) {
                match sys.process(Pid::from(pid)) {
                    Some(process) => Ok(process.kill()),
                    None => Err(Error::ProcessNotRunning(pid)),
                }
            }
```

Reply:
Thanks @GuillaumeGomez. A bit late for accepting the suggestion here so I'll create a commit with your name on it, unless you want to submit a PR? ;)

Reply:
I'll submit a PR then. ;)